test/e2e/e2e.go

```
@@ -697,7 +707,7 @@ spec:
 		createGenericPodWithLabel(f, pod1Name, pod1Node.name, f.Namespace.Name, command, podEgressLabel)
 		createGenericPodWithLabel(f, pod2Name, pod2Node.name, f.Namespace.Name, command, podEgressLabel)
 
-		wait.PollImmediate(retryInterval, retryTimeout, func() (bool, error) {
```

be careful not swallowing these error